### DISCUSSION TITLE
📱📱Please tell me your favorite smartphone app!!!📱📱

### DISCUSSION THREAD

**Discussion by Ota1022** [NAIST-SE/AttractingContributors#142] [General]
https://github.com/Ota1022/AttractingContributors-SmartPhone_App

**Always welcome your Pull requests 👊**

**Comment by zhangyongxin1212** [comment]
Added, plz help me too
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Reply by Ota1022** [reply]
I added WInning Eleven, thank you.

**Comment by MatsumuraTaiki** [comment]
Added. Please help me too.
https://github.com/MatsumuraTaiki/AttractingContributors-Actors

**Reply by Ota1022** [reply]
It seems that I have already added "Zendaya" to your page. 
When you merge my request you might edit twice(if it was my fault I'd like to apologize in advance) so you should delete 4 or 6.

**Reply by MatsumuraTaiki** [reply]
I'm sorry, I didn't realize.
The sixth on the list, Zendaya Maree Stoermer Coleman, was added by another person.

**Reply by Ota1022** [reply]
So many Zendaya fans in the world:)
To avoid overlap I changed mine to "Saoirise Ronan" and send a request again. Please check it.

**Reply by MatsumuraTaiki** [reply]
I appreciate your kindness!!!

**Comment by nagata-atsuki** [comment]
added. Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Reply by Ota1022** [reply]
I added Yusuke Oyama, thank you.

**Comment by ElefunAi** [comment]
I have added App list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Reply by Ota1022** [reply]
I added Xcode, thank you.

**Comment by ellielo** [comment]
The apps i use daily are:
 Instagram
Duolingo - started to learn French recently
[Teensafe](https://theteensafe.com) - phone tracker for my younger brother (he is a pain in the ... lately)
Uber/uber eats
currency converter
pinterest
booking.com
spotify
QR code
Can't say they are my favorite. it is just the ones i use every single day

**Comment by hernessteve** [comment]
I have few of those: Instagram, TikTok, Facebook, GoogleDrive, [mega personal dating app](https://megapersonals.one/mega-personal-dating-app), Telegram, Todolist, FB Messenger. I also use all the food delivery and taxi apps. Spotify for streaming music. Online banking app for obvious reasons. Pretty much all i guess)

**Comment by olehvgrn** [comment]
"Twitter
Pinterest
Uber.
In general, I have never used a taxi, but when I read uber reviews on https://uber.pissedconsumer.com/review.html , I realized that it is not so expensive and convenient. Now, when I'm late somewhere, I immediately call an Uber. Since I am a designer, I am often inspired by photos from Pinterest. I think many people do that too"

**Comment by XelaAl** [comment]
I use various apps (like everyone else, I guess), so it is hard to choose only one. 
Speaking about the applications I use daily, here I can name Telegram, WhatsApp, CoolReader, Instagram, Chrome, Spotify, Google Drive, YouTube, and Google Meet.
